From dd4283e908fcc740005c0f87cf9244eda3b6ba33 Mon Sep 17 00:00:00 2001
From: Tatsuya Matsuo <96454784+matsuo-tat@users.noreply.github.com>
Date: Wed, 17 Jan 2024 09:41:33 +0900
Subject: [PATCH 6/7] [xcvrd] Output CMIS manager log

This change outputs CMIS manager log to console. Users can get these as
container logs when starting xcvrd as a container from the system.
---
 sonic-xcvrd/xcvrd/xcvrd.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sonic-xcvrd/xcvrd/xcvrd.py b/sonic-xcvrd/xcvrd/xcvrd.py
index 17bae9b..b8d6450 100644
--- a/sonic-xcvrd/xcvrd/xcvrd.py
+++ b/sonic-xcvrd/xcvrd/xcvrd.py
@@ -900,10 +900,10 @@ class CmisManagerTask(threading.Thread):
         self.namespaces = namespaces
 
     def log_notice(self, message):
-        helper_logger.log_notice("CMIS: {}".format(message))
+        helper_logger.log_notice("CMIS: {}".format(message), also_print_to_console=True)
 
     def log_error(self, message):
-        helper_logger.log_error("CMIS: {}".format(message))
+        helper_logger.log_error("CMIS: {}".format(message), also_print_to_console=True)
 
     def on_port_update_event(self, port_change_event):
         if port_change_event.event_type not in [port_change_event.PORT_SET, port_change_event.PORT_DEL]:
-- 
2.20.1

